.github/workflows/tests.yml

```diff
@@ -21,11 +21,18 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox tox-factor
+        pip install tox tox-factor coverage
     - name: Lint with flake8
       run: tox -e flake8
       if: matrix.python-version == '3.8'
     - name: Run tests via tox
       run: |
         export TOXFACTOR=$(echo "py${{ matrix.python-version }}" | sed 's/\.//g')
         tox
+    - name: Generate coverage report
```

Comment:
Like in the other PR, `if: ${{ success() }}` could be useful for the coverage steps.